### ISSUE TITLE
Getting unknown property: bedezign\yii2\audit\Audit::tracePathMappings

### ISSUE THREAD

**Issue by GhaziTriki** [bedezign/yii2-audit#270]
I have thee following stack trace when I try to access a trail logs.

```
yii\base\UnknownPropertyException: Getting unknown property: bedezign\yii2\audit\Audit::tracePathMappings in /app/vendor/yiisoft/yii2/base/Component.php:154
Stack trace:
#0 /app/vendor/yiisoft/yii2/di/ServiceLocator.php(77): yii\base\Component->__get('tracePathMappin...')
#1 /app/vendor/yiisoft/yii2-debug/src/Panel.php(144): yii\di\ServiceLocator->__get('tracePathMappin...')
#2 /app/vendor/yiisoft/yii2-debug/src/views/default/panels/log/detail.php(148): yii\debug\Panel->getTraceLine(Array)
#3 [internal function]: yii\base\View::{closure}(Array, 0)
#4 /app/vendor/yiisoft/yii2/helpers/BaseHtml.php(1130): call_user_func(Object(Closure), Array, 0)
#5 /app/vendor/yiisoft/yii2-debug/src/views/default/panels/log/detail.php(149): yii\helpers\BaseHtml::ul(Array, Array)
#6 [internal function]: yii\base\View::{closure}(Array, 1, 1, Object(yii\grid\DataColumn))
#7 /app/vendor/yiisoft/yii2/grid/DataColumn.php(230): call_user_func(Object(Closure), Array, 1, 1, Object(yii\grid\DataColumn))
#8 /app/vendor/yiisoft/yii2/grid/DataColumn.php(244): yii\grid\DataColumn->getDataCellValue(Array, 1, 1)
#9 /app/vendor/yiisoft/yii2/grid/Column.php(111): yii\grid\DataColumn->renderDataCellContent(Array, 1, 1)
#10 /app/vendor/yiisoft/yii2/grid/GridView.php(525): yii\grid\Column->renderDataCell(Array, 1, 1)
#11 /app/vendor/yiisoft/yii2/grid/GridView.php(494): yii\grid\GridView->renderTableRow(Array, 1, 1)
#12 /app/vendor/yiisoft/yii2/grid/GridView.php(358): yii\grid\GridView->renderTableBody()
#13 /app/vendor/yiisoft/yii2/widgets/BaseListView.php(160): yii\grid\GridView->renderItems()
#14 /app/vendor/yiisoft/yii2/grid/GridView.php(326): yii\widgets\BaseListView->renderSection('{items}')
#15 /app/vendor/yiisoft/yii2/widgets/BaseListView.php(135): yii\grid\GridView->renderSection('{items}')
#16 [internal function]: yii\widgets\BaseListView->yii\widgets\{closure}(Array)
#17 /app/vendor/yiisoft/yii2/widgets/BaseListView.php(138): preg_replace_callback('/{\\w+}/', Object(Closure), '{summary}\n{item...')
#18 /app/vendor/yiisoft/yii2/grid/GridView.php(301): yii\widgets\BaseListView->run()
#19 /app/vendor/yiisoft/yii2/base/Widget.php(140): yii\grid\GridView->run()
#20 /app/vendor/yiisoft/yii2-debug/src/views/default/panels/log/detail.php(15): yii\base\Widget::widget(Array)
#21 /app/vendor/yiisoft/yii2/base/View.php(348): require('/app/vendor/yii...')
#22 /app/vendor/yiisoft/yii2/base/View.php(257): yii\base\View->renderPhpFile('/app/vendor/yii...', Array)
#23 /app/vendor/yiisoft/yii2/base/View.php(156): yii\base\View->renderFile('/app/vendor/yii...', Array, NULL)
#24 /app/vendor/bedezign/yii2-audit/src/panels/LogPanel.php(38): yii\base\View->render('@yii/debug/view...', Array)
#25 /app/vendor/bedezign/yii2-audit/src/views/entry/view.php(96): bedezign\yii2\audit\panels\LogPanel->getDetail()
#26 /app/vendor/yiisoft/yii2/base/View.php(348): require('/app/vendor/bed...')
#27 /app/vendor/yiisoft/yii2/base/View.php(257): yii\base\View->renderPhpFile('/app/vendor/bed...', Array)
#28 /app/vendor/yiisoft/yii2/base/View.php(156): yii\base\View->renderFile('/app/vendor/bed...', Array, Object(bedezign\yii2\audit\controllers\EntryController))
#29 /app/vendor/yiisoft/yii2/base/Controller.php(386): yii\base\View->render('view', Array, Object(bedezign\yii2\audit\controllers\EntryController))
#30 /app/vendor/bedezign/yii2-audit/src/controllers/EntryController.php(63): yii\base\Controller->render('view', Array)
#31 [internal function]: bedezign\yii2\audit\controllers\EntryController->actionView('102', 'audit/log')
#32 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#33 /app/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#34 /app/vendor/yiisoft/yii2/base/Module.php(528): yii\base\Controller->runAction('view', Array)
#35 /app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('audit/entry/vie...', Array)
#36 /app/vendor/yiisoft/yii2/base/Application.php(386): yii\web\Application->handleRequest(Object(yii\web\Request))
#37 /app/backend/web/index.php(17): yii\base\Application->run()
#38 {main}
```

**Comment by thinkwcz** [comment]
yiisoft/debug after  2.1.15 New parameter added $tracePathMappings,You can install a lower version.

**Comment by thinkwcz** [comment]
@GhaziTriki I'm sorry I got the wrong version,After version 2.1.5.i use 

composer require --prefer-dist yiisoft/yii2-debug "<=2.1.5"

**Comment by Blizzke** [comment]
I'll release a fix shortly.

**Comment by Blizzke** [comment]
Fixed in 1.1.2, please reopen if this wasn't solved.
